runtime/doc/options.txt

```diff
@@ -2299,7 +2299,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	directory.  If you switch this option on you should also consider
 	setting the 'secure' option (see |initialization|).  Using a local
 	.exrc, .nvimrc or .ngvimrc is a potential security leak, use with care!
```

Comment:
I agree, although perhaps `ex_cmds2.c:2415` should be updated too. In `do_source()`, the `check_other` parameter is true for a few calls, although I'm not sure if that block is needed, as I thought nvim doesn't check for `.nvimrc` files; perhaps it's for Windows?